Modules/Sources/Yosemite/Tools/Plugins/PluginsService.swift

```diff
@@ -5,7 +5,12 @@ import protocol Storage.StorageManagerType
 /// A service for system plugins.
 public protocol PluginsServiceProtocol {
     /// Waits for a specific plugin to be available in storage.
-    func waitForPluginInStorage(siteID: Int64, pluginName: String, isActive: Bool) async -> SystemPlugin
+    /// - Parameters:
+    ///   - siteID: The site ID to search for the plugin.
+    ///   - plugin: The plugin's file path (e.g., "woocommerce/woocommerce.php" for WooCommerce).
+    ///   - isActive: Whether to wait for the plugin to be active or inactive.
```

Comment:
nit: I think the usage of "wait for" bit here could be miss leading, since we're not waiting but matching the current plugin state in storage. wdyt?

```suggestion
    ///   - isActive: Whether to match a plugin that is currently active or inactive.
```

Reply:
Thanks for catching this inaccurate comment, updated the comment in https://github.com/woocommerce/woocommerce-ios/pull/15871/commits/5f83b824d005582fba1c9ff8621ed597ab1787d2.